plutus-core/testlib/PlutusIR/Test.hs

```diff
@@ -97,23 +98,26 @@ goldenPirM :: forall a b . Pretty b => (a -> IO b) -> Parser a -> String -> Test
 goldenPirM op parser name = withGoldenFileM name parseOrError
     where
         parseOrError :: T.Text -> IO T.Text
-        parseOrError =
+        parseOrError txt =
             let parseTxt :: T.Text -> Either ParserErrorBundle a
-                parseTxt txt = runQuoteT $ parse parser name txt
+                parseTxt ptxt = runQuoteT $ parse parser name ptxt
             in
-            either (return . T.pack . show) (fmap display . op)
-                         . parseTxt
+            case parseTxt txt of
+                Left (ParseErrorB err) -> return $ T.pack $  errorBundlePretty err
```

Comment:
The `Pretty` instance for `ParseErrorB` already uses `errorBundlePretty`. So you don't need to do this, you can just change the original Left case to use `display` also.

Reply:
i.e.
```
either (return . display) (fmap display . op)
```